ios/FluentUI/Other Cells/ActionsCell.swift

```diff
@@ -186,6 +184,14 @@ open class ActionsCell: UITableViewCell, TokenizedControlInternal {
             verticalSeparator.frame = CGRect(x: left, y: 0, width: verticalSeparator.frame.width, height: frame.height)
         }
 
+        // A hacky way to hide the system separator by squeezing it just enough to have zero width.
+        // This is the only known way to hide the separator without making the UITableView do it for us.
```

Comment:
its a legacy behavior since the pre fluent1 days (aka fabric ios). lots of client code need to update their tableview to not have separators... which lead us to have this hack...

Reply:
oh sorry I meant the comment is kinda confusing

Reply:
I mean, yes, `UITableView` can hide separators depending on its configuration. But what we need here is for the cell itself to always hide the system separator, regardless of the table view configuration.

Reply:
So I think you interpreted it correctly? Or I'm misinterpreting your interpretation :D